cylc/flow/task_job_mgr.py

```diff
@@ -287,7 +287,13 @@ def submit_task_jobs(self, suite, itasks, curve_auth,
                 cmd.append('--debug')
             if get_utc_mode():
                 cmd.append('--utc-mode')
-            remote_mode = False
+            if is_remote_platform(itask.platform):
+                remote_mode = True
+                cmd.append('--remote-mode')
+            else:
+                remote_mode = False
+            # TODO - check whether Kwargs is actually used in
```

Comment:
Is this a TODO to sort at a later date or was this a reminder for this/platforms pr?

Reply:
I think I'd rather not make any _more_ changes, but I think that it's worth checking if this simplification can be made as a stand-alone PR later.